modules/nw-cluster-mtu-change-about.adoc

```diff
@@ -30,6 +30,7 @@ When planning your MTU migration there are two related but distinct MTU values t
 
 * *Hardware MTU*: This MTU value is set based on the specifics of your network infrastructure.
 * *Cluster network MTU*: This MTU value is always less than your hardware MTU to account for the cluster network overlay overhead. The specific overhead is determined by your network plugin. For OVN-Kubernetes, the overhead is `100` bytes.
+* *OpenShift-SDN*: The specific overhead is determined by your network plugin. The overhead is `50` bytes.
```

Comment:
Shouldn't this be incorporated in the previous bullet?

Reply:
I agree. Updated.